### ISSUE TITLE
Conflict with MbedTLS

### ISSUE THREAD

**Issue by jeremiedb** [aviks/SMTPClient.jl#36]
There seems to be a conflict using SMTPClient along with MbedTLS, at least on Windows 10. 

If calling SSLConfig() from MbedTLS, and no other library is loaded, there's no issue: 

```
julia> using MbedTLS: SSLConfig
julia> SSLConfig(false)
MbedTLS.SSLConfig()
```

If SMTPClient is also used, `SSLConfig` crashes 

```
julia> using MbedTLS: SSLConfig
julia> using SMTPClient
julia> SSLConfig(false)
ERROR: error compiling SSLConfig: error compiling #SSLConfig#46: could not load library "C:\Users\Jeremie\.julia\packages\MbedTLS\a1JFn\deps\usr\bin\libmbedtls.dll"
The specified procedure could not be found.

Stacktrace:
 [1] top-level scope at none:0
```

Also, if `SSLConfig` is called prior to having SMTPClient loaded, then calls to `SSLConfig` will continue to work after SMTPClient is loaded. 

SMTPClient version: 0.3.1
MbedTLS version: 0.7.0
Removing and reinstalling plus build has been attempted. 

Any insight why these libraries would conflict?

**Comment by iblislin** [comment]
cc me

**Comment by aviks** [comment]
Fixed by #39
